protocol/common_types.go

```diff
@@ -146,6 +146,8 @@ func (h ByteSlice) String() string {
 	return "0x" + hex.EncodeToString(h)
 }
 
+type Bytes16 [16]byte
```

Comment:
Do we not want the MarshalJSON/String et al helpers so this can show up nicely in logs?